src/api/config/routes.rb

```diff
@@ -304,7 +304,7 @@ def self.public_or_about_path?(request)
       get 'request/diff/:number' => :diff
       get 'request/list_small' => :list_small
       get 'request/delete_request_dialog' => :delete_request_dialog
-      post 'request/delete_request' => :delete_request
+      post 'request/delete_request/:project' => :delete_request, as: :request_delete, constraints: cons
```

Comment:
why do you need a name for this action? :thinking:

Reply:
@Ana06 

> why do you need a name for this action? :thinking:

To use the route helper `request_delete`in the partial `request/_delete_request_dialog.html.erb`

A default route helper is not created in this case.

Reply:
Yes, but why is do you want to have a route helper? I am not sure if giving random names to routes is something useful.

And should it be `delete_request` instead `request_delete`? Both the route and controller method are called like that. :thinking:

Reply:
@Ana06 

> Yes, I know that. :unamused: but why do you want to have a route helper?

It has better readibility.

> I am not sure if giving random names to routes is something useful.

Sorry. :hankey:

> And should it be `delete_request` instead `request_delete`? Both the route and controller method are called like that. :thinking:

Explained by examples:
- `project/rebuild_time/:project/:repository/:arch` -> `project_rebuild_time`
- `project/subprojects/:project` -> `project_subprojects`

So for this one it should be:
- `request/delete_request/:project` -> `request_delete_request`

But as it seemed redundant `request_delete` was changed to `request_delete`

In fact these two routes use `_request` at the end of the action name, something for a deeper refactoring:

```
get 'request/delete_request_dialog' => :delete_request_dialog
post 'request/delete_request/:project' => :delete_request, as: :request_delete, constraints: cons
```

Should I get rid of this route helper then?

Reply:
@eduardoj 

> It has better readibility.

Why is it more readable? When I find that route in the code I need to go to the routes files because it is not evident what controller and action I am being redirected. So, what you call readability, IMHO can be better call obfuscating code. :wink: 

And yes, you are right that we are doing some strange things with the routes that should be fixed. I think that this PR is not the place to do so, but at least do not introduce more strange things :rofl: